spec/modules/batch_spec.rb

```diff
@@ -61,7 +61,7 @@
   end
 
   context 'flushing' do
-    it 'must put wokrer to queue on flush' do
+    it 'must put worker to queue on flush' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.